static/src/javascripts/projects/commercial/modules/liveblog-adverts.spec.js

```diff
@@ -63,21 +75,23 @@ describe('Liveblog Dynamic Adverts', () => {
 		expect(desktopSlot).toBe('inline1');
 	});
 
-	// todo: difficult to mock spacefiller, which is not yet ES6'ed, so come back to this
-	it.skip('should insert ads every 5th block', () =>
-		init().then(() => {
-			const adSlots = document.querySelectorAll(
-				'.js-liveblog-body .ad-slot',
-			);
-			const candidates = document.querySelectorAll(
-				'.js-liveblog-body > *:nth-child(5n+1)',
+	it('should insert ad slots', async () => {
+		spaceFiller.fillSpace.mockImplementationOnce(
+			createFillSpaceMock([
+				document.querySelector('.x1'),
+				document.querySelector('.x12'),
+			]),
+		);
```

Comment:
fyi I added this `spaceFiller` mock which might be useful for other `spaceFiller` clients.